src/components/newUIComponents/overview/TimeIncrementSelector.js

```diff
@@ -0,0 +1,50 @@
+import { Menu, MenuItem, MenuButton } from "@szhsin/react-menu";
+import { FaChevronDown } from "react-icons/fa";
+
+export default function TimeIncrementSelector({
+  timeIncrement,
+  setTimeIncrement,
+}) {
+  const timeIncrementToText = (timeIncrement) => {
+    switch (timeIncrement) {
```

Comment:
Should use constants for these (or define new constants and use those so we don't have 'magic numbers' when used in util methods too